### PR TITLE
[new release] ocal (0.2.2)

### PR DESCRIPTION
**Pull request by mor1**
An improved Unix `cal` utility

- Project page: <a href="https://github.com/mor1/ocal">https://github.com/mor1/ocal</a>
- Documentation: <a href="https://mor1.github.io/ocal/">https://mor1.github.io/ocal/</a>

##### CHANGES:

* Dunification

### CODE REVIEW

File: packages/ocal/ocal.0.2.2/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+bug-reports: "https://github.com/mor1/ocal/issues"
+doc: "https://mor1.github.io/ocal/"
+
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+run-test: [
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
Comment (kit-ty-kate) [inline packages/ocal/ocal.0.2.2/opam]: This doesn't work. It should be either tagged with `{with-test}` inside `build` or without inside `run-test`.

**Comment by kit-ty-kate** [comment]
Fixed. Could you return the fix upstream?

File: packages/ocal/ocal.0.2.2/opam [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+]
+
+depends: [
+  "astring"   {build}
+  "calendar"  {build}
+  "cmdliner"  {build}
+  "dune"      {build}
+  "notty"     {build & >="0.2.0"}
+  "ocaml"     {build & >="4.02.3"}
```
Comment (kit-ty-kate) [inline packages/ocal/ocal.0.2.2/opam]: mmh, something tells me `ocaml` shouldn't be a build dependency. What if `dune` decides to compile the executable to bytecode if the platform doesn't support native code?

Comment (mor1) [reply packages/ocal/ocal.0.2.2/opam]: This is as recommended by `dune-release lint`:
```
: mort@greyjay:ocal#master= $; make lint
dune build @lint
dune-release lint
[FAIL] lint opam file ocal.opam:
       opam lint -s messages:
       /Users/mort/src/ocal/ocal.opam: Errors.
                    error 32: Field 'ocaml-version:' and variable 'ocaml-version' are deprecated, use a dependency towards the 'ocaml' package instead for availability, and the 'ocaml:version' package variable for scripts
[FAIL] lint /Users/mort/src/ocal failure: 1 errors.
make: *** [lint] Error 1
```
(Or at least, that's how I interpreted the error message! Am I supposed to do something else here?)

File: packages/ocal/ocal.0.2.2/opam (hunk 1 above)
Comment (mor1) [reply packages/ocal/ocal.0.2.2/opam]: Ah ok -- I think this was me misinterpreting the error from `dune-release lint` indicating what to do to remove the `build-test` field. Will fix!

File: packages/ocal/ocal.0.2.2/opam (hunk 2 above)
Comment (kit-ty-kate) [reply packages/ocal/ocal.0.2.2/opam]: oh sorry, I should have been more clear. I meant to insist on the `build` dependency part, not the `dependency` part of the sentence ^^

OCaml should of course be a dependency but not flagged as a `{build}` dependency probably. Anyone has a thought on that btw? @rjbou @rgrinberg?

Comment (rgrinberg) [reply packages/ocal/ocal.0.2.2/opam]: Setting `ocaml` as `{build}` seems wrong. At the very least, the compiler contains pervasives and it's highly unlikely that Pervasives are only used for building a package.

**Comment by kit-ty-kate** [comment]
Sorry I just noticed the thing about the unneeded ocamlfind dependency. Could you return the fix upstream?

I'll merge it tomorrow

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**